ansible_collections/arista/avd/tests/integration/targets/yaml_templates_to_facts/tests/test_with_list_merge_append_rp.yml

```diff
@@ -1,29 +0,0 @@
-- name: Test with list_merge append_rp option
```

Comment:
Instead of removing these tests, we should implement support for those merge methods. I suggest we move the merge into a separate library where we can load the relevant strategies - thereby simplifying the code in yaml_templates_to_facts.